src/components/Questions/Questions.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import QuestionImage from './../../assets/icons/QuestionImages.svg';
+import Answers from './../Answers/Answers';
+function Questions() {
+  return (
+    <div className="flex flex-col bg-darkgray w-full justify-center items-center">
+      <div className="imgWrapper flex ">
+        <img src={QuestionImage} alt="question" className="flex" />
+      </div>
+      <div className="flex content-center text-white p-5">
+        <span className="sm:text-2xl md:text-2xl text-3xl md:px-24 xs:text-xl lg:px-32 xl:px-48 text-center">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Earum nihil?
```

Comment:
you should get this from the question prop instead of writing it
```suggestion
     {question.title}
```